pontoon/base/models.py

```diff
@@ -1,11 +1,15 @@
+from __future__ import unicode_literals
 from __future__ import division
+from django.utils.encoding import python_2_unicode_compatible
```

Comment:
@adngdb Should be put this into the [CONTRIBUTING](https://github.com/mozilla/pontoon/blob/master/CONTRIBUTING.rst) file?